icons/chains/white/Shiden.svg

```diff
@@ -1,9 +1,10 @@
+      <path d="M45.0292 6.33881C44.996 6.34232 41.612 6.70246 36.8976 8.04512C31.9082 9.46678 24.643 12.2234 18.3696 17.2806C14.6704 20.1544 12.3291 23.6214 11.9458 26.7932C11.7578 28.3411 13.5719 31.6268 13.5719 31.6268L13.1905 30.4597C13.1731 30.4066 11.552 25.0907 19.7668 18.9227C24.3698 15.4659 29.1668 13.1001 34.8619 11.4779C37.7979 10.6415 40.6111 10.0921 43.7209 9.51092L43.8304 9.49045L43.8779 9.38906C44.1128 8.88533 45.2331 6.67943 45.2448 6.65736L45.4265 6.29819L45.0292 6.33881Z" fill="#764BC4"></path>
+      <path d="M9.07686 22.4783C9.38676 17.4975 13.1588 12.8954 20.2967 8.79544C24.0583 6.4469 28.263 4.60177 32.7938 3.31092C36.6365 2.21613 39.8846 1.88926 39.8846 1.88926L37.2724 0C37.2484 0.00287851 34.8407 0.298725 31.4001 1.27902C9.33801 7.56408 6.60396 18.393 6.84802 22.746C7.27536 30.3353 16.2122 35.7325 16.2122 35.7325C16.2122 35.7325 8.61564 29.8997 9.07686 22.4783Z" fill="#BBA5E1"></path>
+      <path d="M9.35057 45.4388C14.1599 44.0687 21.2564 41.203 27.7568 35.3289C34.2894 29.1807 29.8546 23.6975 29.8087 23.6431C29.8087 23.6431 32.2027 28.0584 26.4124 33.4985C26.3389 33.5628 18.9475 39.9463 8.19769 43.0084C6.82702 43.3989 5.44147 43.7239 4.07935 43.9743L3.9847 43.9916L2.25412 46.7965L2.67893 46.7665C2.70583 46.7642 5.40665 46.5624 9.35057 45.4388Z" fill="#5928B1"></path>
+      <path d="M1.95225 53.2997C3.05955 54.2317 4.16558 54.9693 4.17666 54.9766L4.24345 55.0201L4.32196 55.0108C4.35171 55.007 7.33584 54.6398 11.4456 53.4689C18.7988 51.3746 28.6879 47.0793 34.0427 38.5157C36.5862 34.6256 34.539 29.971 34.539 29.971L34.6007 31.1557C34.6074 31.2814 34.9588 43.8304 9.23579 51.1588C6.98416 51.8003 4.56602 52.3914 2.04722 52.9172L1.60626 53.009L1.95225 53.2997Z" fill="#764BC4"></path>
+      <path d="M0.397293 51.6612C0.430531 51.6577 3.81447 51.2976 8.52887 49.9549C13.5184 48.5332 20.7835 45.7766 27.0569 40.7194C30.7562 37.8457 33.0974 34.3787 33.4807 31.2069C33.6687 29.6589 31.8546 26.3732 31.8546 26.3732L32.236 27.5403C32.2534 27.5934 33.8745 32.9094 25.6597 39.0774C21.0567 42.5341 16.2597 44.8999 10.5646 46.5221C7.6286 47.3585 4.8154 47.908 1.7056 48.4891L1.59607 48.5096L1.54859 48.611C1.31371 49.1147 0.193434 51.3206 0.181722 51.3427L2.21315e-05 51.7018L0.397293 51.6612Z" fill="#764BC4"></path>
+      <path d="M36.3497 35.5217C36.0398 40.5025 32.2677 45.1046 25.1298 49.2046C21.3682 51.5531 17.1635 53.3982 12.6327 54.6891C8.79006 55.7839 5.54192 56.1107 5.54192 56.1107L8.15411 58C8.17816 57.9971 10.5859 57.7013 14.0264 56.721C36.0885 50.4359 38.8226 39.607 38.5785 35.254C38.1512 27.6647 29.2143 22.2675 29.2143 22.2675C29.2143 22.2675 36.8109 28.1003 36.3497 35.5217Z" fill="#BBA5E1"></path>
+    </g>
```

Comment:
Are you sure svg images can start not with svg tag?

Reply:
![image](https://user-images.githubusercontent.com/70131744/134697324-9c1d5e61-4d9e-4b11-8ed8-096986dab190.png)
Github cannot decode it

Reply:
fixed